pom.xml

```diff
@@ -95,43 +95,14 @@
       <name>Sonatype Staging Repository</name>
       <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
     </repository>
-    <snapshotRepository>
-      <id>oss-sonatype-snapshots</id>
-      <name>OSS Sonatype Nexus Snapshots</name>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-    </snapshotRepository>
   </distributionManagement>
 
   <repositories>
-    <repository>
-      <id>oss-sonatype-snapshots</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
-      <name>OSS Sonatype Nexus Snapshots</name>
-      <releases>
-        <enabled>false</enabled>
-      </releases>
-      <snapshots>
-        <enabled>true</enabled>
-      </snapshots>
-    </repository>
     <repository>
       <id>jenkins</id>
       <url>http://repo.jenkins-ci.org/releases/</url>
     </repository>
   </repositories>
-  <pluginRepositories>
-    <pluginRepository>
-      <id>oss-sonatype-snapshots</id>
```

Comment:
Are you sure about removing content repositories?
I think we should remove it. Since it could be used for releasing fabric8 pieline library. Definitely, it has no relation to booster build content repository.